File: JessicaKang/week_08/Burning_Airlines/Burning_Airlines/app/assets/javascripts/models/Flight.js
var app = app || {};

app.Flight = Backbone.Model.extend({
  urlRoot: "/flights",

  defaults: {
    flight_number: "",
    origin: "",
    destination: "",
    date: ""
  },

  initialize: function () {
    console.log( "A new flight was created" );
  }
});

File: JessicaKang/week_08/Burning_Airlines/Burning_Airlines/app/models/flight.rb
# == Schema Information
#
# Table name: flights
#
#  id            :integer          not null, primary key
#  flight_number :text
#  origin        :text
#  destination   :text
#  date          :datetime
#  plane         :text
#  created_at    :datetime         not null
#  updated_at    :datetime         not null
#

class Flight < ActiveRecord::Base
  has_many :reservations
  belongs_to :airplane
  has_many :users, :through => :reservations
end
